### PR TITLE
v1.11.0

### PR DESCRIPTION
**Pull request by robframpton**
### Added

- Added MFA API support (https://github.com/workos/workos-python/pull/116)

### CODE REVIEW

**Review by awolfden — APPROVED** [review]

**Comment by robframpton** [comment]
@awolfden have you seen these CI errors before? It's strange that it's passing on `master` but failing with this simple version bump.

**Comment by awolfden** [comment]
once this PR is approved it should pass CI after merging in the changes - https://github.com/workos/workos-python/pull/120

**Pull request merged**
